test/system/700-play.bats

```diff
@@ -182,8 +182,10 @@ EOF
     run_podman container inspect --format "{{.HostConfig.NetworkMode}}" $infraID
     is "$output" "none" "network mode none is set for the container"
 
-    run_podman stop -a -t 0
-    run_podman pod rm -t 0 -f test_pod
+   run_podman kube down - < $PODMAN_TMPDIR/test.yaml
+   run_podman 125 inspect test_pod-test
```

Comment:
Please add a check for output. Exit-status-only tests are evil. The test above could just as easily fail with a typo:
```
run_podman 125 ipsnect blah blah
```
You need to actually confirm something, like `assert "$output" = "[]\nError: inspecting object: no such container test_pod-test"`. (And BTW, does anyone know why `podman inspect` spits out that confusing `[]`?)

Same in line 358.

Reply:
Ah, probably worth doing a `podman pod inspect`.

>  (And BTW, does anyone know why podman inspect spits out that confusing []?)

I never noticed but will take a look.

Reply:
This is for docker compat AFAIK, so that you can still parse json correctly.